Client/U2FExtensions.swift

```diff
@@ -319,6 +374,33 @@ class U2FExtensions: NSObject {
         }) }
     }
     
+    private func handleMakeCredential(handle: Int, request: WebAuthnRegisterRequest, error: Error?) {
+        guard let error = error else {
```

Comment:
Ideally error should not be `nil` here see: https://github.com/brave/brave-ios/pull/1303/files#diff-1faf1a9d0dd722c9a8d957734c58b2e6R333 - I'll update the code to pass `Error` instead of `Error?` here.